tests/test_cirq_interface.py

```diff
@@ -66,6 +66,19 @@ def test_apply(self):
         assert gate_applications[3] == cirq.Z.on(qubit)
         assert gate_applications[4] == cirq.Rz(0.3).on(qubit)
 
+    def test_apply_not_parametrized(self):
+        """Tests that the proper error is raised if an Operation is applied
+        that was not parametrized before."""
+        operation = CirqOperation(
+            lambda a, b, c: [cirq.X, cirq.Ry(a), cirq.Rx(b), cirq.Z, cirq.Rz(c)]
+        )
+        qubit = cirq.LineQubit(1)
+
+        with pytest.raises(
+            qml.DeviceError, match="CirqOperation must be parametrized before it can be applied."
+        ):
+            operation.apply(qubit)
```

Comment:
Maybe I'm just missing context, but I'm a bit confused what this is testing?

Reply:
The `CirqOperation` class encodes my mapping between Cirq operations and PennyLane gates. For example, `Rot` maps to 
```python
"Rot": CirqOperation(lambda a, b, c: [cirq.Rz(a), cirq.Ry(b), cirq.Rz(c)])
```
which means the successive application of `Rz`, `Ry` and `Rz`. This also needs to be parametrized with the parameters of `Rot`. 

The test checks that an error is raised if a non-parametrized operation is applied to a qubit.